pkg/virt-api/webhooks/validating-webhook/admitters/vmclone-admitter_test.go

```diff
@@ -359,6 +359,34 @@ var _ = Describe("Validating VirtualMachineClone Admitter", func() {
 		)
 	})
 
+	Context("VirtualMachineTemplate Annotations and labels filters", func() {
+		templateTestFilter := func(filter string, expectAllowed bool) {
+			vmClone.Spec.Template.LabelFilters = []string{filter}
+			vmClone.Spec.Template.AnnotationFilters = []string{filter}
+			admitter.admitAndExpect(vmClone, expectAllowed)
+		}
+
+		DescribeTable("Should reject", func(filter string) {
+			templateTestFilter(filter, false)
```

Comment:
You could call both `testFilter` and `templateTestFilter` in the same table to keep down duplication of test cases?